{libbsd,libmd}: update

emilazy opened the pull request:
## Description of changes

The current Git revision of `libbsd` has had a lot of work done to it to better support macOS and lets us drop the huge patch we were using before. More macOS fixes are apparently on the way, so hopefully the two that are still required can be dropped on the next release.

This got https://github.com/NixOS/nixpkgs/pull/240405 building and running successfully for me. I've tested all downstream packages with no build regressions on `x86_64-darwin`, but would appreciate Linux and Apple Silicon users running `nixpkgs-review`.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

emilazy committed:
libmd: 1.0.4 -> 1.1.0

The symbol alias macros now support macOS, so the patch can be dropped.

emilazy committed:
libbsd: 0.11.7 -> unstable-2023-04-29

The current Git revision has had a lot of work done to it to
better support macOS and lets us drop the huge patch we were using
before. More macOS fixes are apparently on the way, so hopefully the
two that are still required can be dropped on the next release.

emilazy committed:
inputplug: mark broken on Darwin

emilazy committed:
libmd: enable tests

emilazy committed:
libbsd: enable tests

tjni commented:
Result of `nixpkgs-review pr 247082` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>inputplug</li>
    <li>netcat-openbsd</li>
  </ul>
</details>
<details>
  <summary>39 packages built:</summary>
  <ul>
    <li>acme-client</li>
    <li>bdf2psf</li>
    <li>cargo-deb</li>
    <li>csmith</li>
    <li>debian-devscripts</li>
    <li>donkey</li>
    <li>dpkg</li>
    <li>enum4linux</li>
    <li>enum4linux-ng</li>
    <li>enum4linux-ng.dist</li>
    <li>gohufont</li>
    <li>got</li>
    <li>libbsd</li>
    <li>libbsd.dev</li>
    <li>libbsd.man</li>
    <li>libmd</li>
    <li>opendkim</li>
    <li>php81Extensions.smbclient</li>
    <li>php82Extensions.smbclient</li>
    <li>php83Extensions.smbclient</li>
    <li>python310Packages.pysmbc</li>
    <li>python310Packages.pysmbc.dist</li>
    <li>python311Packages.pysmbc</li>
    <li>python311Packages.pysmbc.dist</li>
    <li>rcon</li>
    <li>rosie</li>
    <li>rs</li>
    <li>samba</li>
    <li>samba.dev</li>
    <li>samba.man</li>
    <li>samba4Full</li>
    <li>samba4Full.dev</li>
    <li>samba4Full.man</li>
    <li>stress-ng</li>
    <li>unifi</li>
    <li>unifi5</li>
    <li>unifi6</li>
    <li>unifiLTS</li>
    <li>uw-ttyp0</li>
  </ul>
</details>

emilazy commented:
(Force pushed just to enable tests for both packages; shouldn't affect anything ofborg won't test.)

pbsds commented on pkgs/development/libraries/libbsd/default.nix:
```diff
@@ -1,28 +1,48 @@
-  patches = [ ./darwin.patch ];
+  patches = [
+    # Temporary fix to make `src/progname.c` compile on Darwin
+    ./darwin-fix-getprogname.patch
```
Any specific reason this is not inside the darwin optional?

emilazy replied:
It should be correct for all platforms (unlike the macOS-specific hack in the other patch), and on Linux it's a no-op, so I lean towards avoiding source code divergence between platforms where possible (e.g., if I think it would be reasonable for it to be committed upstream theres no reason to make it conditional; the only reason I'm not PRing this is that apparently there is a bunch of macOS porting work in the pipeline for `libbsd` already so it's likely redundant).

Arguably that means `darwin-fix-getprogname.patch` is a misleading name but I couldn't think of a better one (`fix-getprogname-on-platforms-with-__progname-support.patch` is a bit long...).

drupol commented on pkgs/development/libraries/libmd/default.nix:
```diff
@@ -1,35 +1,30 @@
 { lib, stdenv, fetchurl, fetchpatch, autoreconfHook }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```
:heart:

emilazy commented:
501-1000 rebuilds for Linux is more than I expected. Should this go to `staging` instead? There's no `stdenv` change and I'm guessing that most of the rebuilds are leaf packages, but I can retarget this if desired. (Is there any way to see a list of packages needing rebuild from ofborg?)

reckenrode commented:
My rule of thumb has been 500+ rebuilds → staging.

I’m running nixpkgs-review on both x86_64-linux and aarch64-linux. I’ll post a gist with the list of packages once it gets to the point of printing what it’s going to rebuild.

adamcstephens commented:
Result of `nixpkgs-review pr 247082` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>95 packages marked as broken and skipped:</summary>
  <ul>
    <li>cinny-desktop</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>guestfs-tools</li>
    <li>kcgi</li>
    <li>libvmi</li>
    <li>linuxKernel.packages.hardkernel_4_14.dpdk</li>
    <li>linuxKernel.packages.hardkernel_4_14.dpdk.doc</li>
    <li>linuxKernel.packages.hardkernel_4_14.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_4_14_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_4_14_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_14_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_4_19_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_4_19_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_19_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_10_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_5_10_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_10_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_15_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_5_15_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_15_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_4_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_5_4_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_4_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_6_1_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_6_1_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_6_1_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_6_4_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_6_4_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_6_4_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_lqx.dpdk</li>
    <li>linuxKernel.packages.linux_lqx.dpdk.doc</li>
    <li>linuxKernel.packages.linux_lqx.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_xanmod.dpdk</li>
    <li>linuxKernel.packages.linux_xanmod.dpdk.doc</li>
    <li>linuxKernel.packages.linux_xanmod.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_xanmod_latest.dpdk</li>
    <li>linuxKernel.packages.linux_xanmod_latest.dpdk.doc</li>
    <li>linuxKernel.packages.linux_xanmod_latest.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_xanmod_stable.dpdk</li>
    <li>linuxKernel.packages.linux_xanmod_stable.dpdk.doc</li>
    <li>linuxKernel.packages.linux_xanmod_stable.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_zen.dpdk</li>
    <li>linuxKernel.packages.linux_zen.dpdk.doc</li>
    <li>linuxKernel.packages.linux_zen.dpdk.kmod</li>
    <li>linuxPackages_4_14_hardened.dpdk</li>
    <li>linuxPackages_4_14_hardened.dpdk.doc</li>
    <li>linuxPackages_4_14_hardened.dpdk.kmod</li>
    <li>linuxPackages_4_19_hardened.dpdk</li>
    <li>linuxPackages_4_19_hardened.dpdk.doc</li>
    <li>linuxPackages_4_19_hardened.dpdk.kmod</li>
    <li>linuxPackages_5_10_hardened.dpdk</li>
    <li>linuxPackages_5_10_hardened.dpdk.doc</li>
    <li>linuxPackages_5_10_hardened.dpdk.kmod</li>
    <li>linuxPackages_5_15_hardened.dpdk</li>
    <li>linuxPackages_5_15_hardened.dpdk.doc</li>
    <li>linuxPackages_5_15_hardened.dpdk.kmod</li>
    <li>linuxPackages_5_4_hardened.dpdk</li>
    <li>linuxPackages_5_4_hardened.dpdk.doc</li>
    <li>linuxPackages_5_4_hardened.dpdk.kmod</li>
    <li>linuxPackages_6_1_hardened.dpdk</li>
    <li>linuxPackages_6_1_hardened.dpdk.doc</li>
    <li>linuxPackages_6_1_hardened.dpdk.kmod</li>
    <li>linuxPackages_6_4_hardened.dpdk</li>
    <li>linuxPackages_6_4_hardened.dpdk.doc</li>
    <li>linuxPackages_6_4_hardened.dpdk.kmod</li>
    <li>linuxPackages_hardened.dpdk</li>
    <li>linuxPackages_hardened.dpdk.doc</li>
    <li>linuxPackages_hardened.dpdk.kmod</li>
    <li>linuxPackages_hardkernel_latest.dpdk</li>
    <li>linuxPackages_hardkernel_latest.dpdk.doc</li>
    <li>linuxPackages_hardkernel_latest.dpdk.kmod</li>
    <li>linuxPackages_lqx.dpdk</li>
    <li>linuxPackages_lqx.dpdk.doc</li>
    <li>linuxPackages_lqx.dpdk.kmod</li>
    <li>linuxPackages_xanmod.dpdk</li>
    <li>linuxPackages_xanmod.dpdk.doc</li>
    <li>linuxPackages_xanmod.dpdk.kmod</li>
    <li>linuxPackages_xanmod_latest.dpdk</li>
    <li>linuxPackages_xanmod_latest.dpdk.doc</li>
    <li>linuxPackages_xanmod_latest.dpdk.kmod</li>
    <li>linuxPackages_xanmod_stable.dpdk</li>
    <li>linuxPackages_xanmod_stable.dpdk.doc</li>
    <li>linuxPackages_xanmod_stable.dpdk.kmod</li>
    <li>linuxPackages_zen.dpdk</li>
    <li>linuxPackages_zen.dpdk.doc</li>
    <li>linuxPackages_zen.dpdk.kmod</li>
    <li>mailnagWithPlugins</li>
    <li>mailnagWithPlugins.dist</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>rubyPackages_2_7.ruby-libvirt</li>
    <li>wtwitch</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>26 packages failed to build:</summary>
  <ul>
    <li>ceph (ceph-dev)</li>
    <li>ceph-client</li>
    <li>ceph-csi</li>
    <li>ceph.dev (ceph-dev.dev)</li>
    <li>ceph.doc (ceph-dev.doc)</li>
    <li>libceph (ceph.lib ,libceph.dev ,libceph.doc ,libceph.lib ,libceph.man)</li>
    <li>ceph.man (ceph-dev.man)</li>
    <li>pktgen (linuxPackages.pktgen)</li>
    <li>linuxPackages_latest.dpdk (linuxKernel.packages.linux_6_4.dpdk)</li>
    <li>linuxPackages_latest.dpdk.doc (linuxKernel.packages.linux_6_4.dpdk.doc)</li>
    <li>linuxPackages_latest.dpdk.kmod (linuxKernel.packages.linux_6_4.dpdk.kmod)</li>
    <li>linuxPackages_latest-libre.dpdk (linuxKernel.packages.linux_latest_libre.dpdk)</li>
    <li>linuxPackages_latest-libre.dpdk.doc (linuxKernel.packages.linux_latest_libre.dpdk.doc)</li>
    <li>linuxPackages_latest-libre.dpdk.kmod (linuxKernel.packages.linux_latest_libre.dpdk.kmod)</li>
    <li>linuxPackages_testing_bcachefs.dpdk (linuxKernel.packages.linux_testing_bcachefs.dpdk)</li>
    <li>linuxPackages_testing_bcachefs.dpdk.doc (linuxKernel.packages.linux_testing_bcachefs.dpdk.doc)</li>
    <li>linuxPackages_testing_bcachefs.dpdk.kmod (linuxKernel.packages.linux_testing_bcachefs.dpdk.kmod)</li>
    <li>pdfstudioviewer</li>
    <li>peek</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>qemu_full</li>
    <li>qemu_full.ga</li>
    <li>redpanda-server</li>
  </ul>
</details>
<details>
  <summary>432 packages built:</summary>
  <ul>
    <li>MIDIVisualizer</li>
    <li>acme-client</li>
    <li>adapta-gtk-theme</li>
    <li>almanah</li>
    <li>apkg</li>
    <li>apkg.dist</li>
    <li>appgate-sdp</li>
    <li>appvm</li>
    <li>apt</li>
    <li>arcanPackages.arcan</li>
    <li>ffmpeg-full (arcanPackages.ffmpeg ,ffmpeg_5-full)</li>
    <li>ffmpeg-full.bin (arcanPackages.ffmpeg.bin ,ffmpeg_5-full.bin)</li>
    <li>ffmpeg-full.data (arcanPackages.ffmpeg.data ,ffmpeg_5-full.data)</li>
    <li>ffmpeg-full.dev (arcanPackages.ffmpeg.dev ,ffmpeg_5-full.dev)</li>
    <li>ffmpeg-full.doc (arcanPackages.ffmpeg.doc ,ffmpeg_5-full.doc)</li>
    <li>ffmpeg-full.lib (arcanPackages.ffmpeg.lib ,ffmpeg_5-full.lib)</li>
    <li>ffmpeg-full.man (arcanPackages.ffmpeg.man ,ffmpeg_5-full.man)</li>
    <li>arcanPackages.xarcan</li>
    <li>armcord</li>
    <li>aucatctl</li>
    <li>aucatctl.man</li>
    <li>audiobookshelf</li>
    <li>autofs5</li>
    <li>bdf2psf</li>
    <li>blackfire</li>
    <li>bluemail</li>
    <li>bluez-alsa</li>
    <li>booster</li>
    <li>bubblemail</li>
    <li>bubblemail.dist</li>
    <li>budgie.budgie-control-center</li>
    <li>budgie.budgie-control-center.debug</li>
    <li>buildah</li>
    <li>buildah.man</li>
    <li>bumblebee</li>
    <li>calls</li>
    <li>calls.devdoc</li>
    <li>cargo-deb</li>
    <li>chatty</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>cinnamon.nemo</li>
    <li>cinnamon.nemo-fileroller</li>
    <li>cinnamon.nemo-python</li>
    <li>cinnamon.nemo-with-extensions</li>
    <li>cinnamon.nemo.dev</li>
    <li>clementine</li>
    <li>collectd</li>
    <li>corrscope</li>
    <li>corrscope.dist</li>
    <li>cri-o</li>
    <li>cri-o.man</li>
    <li>criu</li>
    <li>criu.dev</li>
    <li>criu.man</li>
    <li>crun</li>
    <li>csmith</li>
    <li>dcp9020cdw-cupswrapper</li>
    <li>debian-devscripts</li>
    <li>debootstrap</li>
    <li>deepin.deepin-music</li>
    <li>deepin.deepin-voice-note</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>distrobuilder</li>
    <li>docker-machine-kvm</li>
    <li>docker-machine-kvm2</li>
    <li>donkey</li>
    <li>dpdk</li>
    <li>dpdk.doc</li>
    <li>dpkg</li>
    <li>eaglemode</li>
    <li>eid-mw</li>
    <li>endeavour</li>
    <li>enum4linux</li>
    <li>enum4linux-ng</li>
    <li>enum4linux-ng.dist</li>
    <li>evolution</li>
    <li>evolution-data-server (gnome.evolution-data-server)</li>
    <li>evolution-data-server-gtk4</li>
    <li>evolution-data-server-gtk4.dev</li>
    <li>evolution-data-server.dev (gnome.evolution-data-server.dev)</li>
    <li>evolution-ews</li>
    <li>far2l</li>
    <li>ffcast</li>
    <li>ffmpeg_4-full</li>
    <li>ffmpeg_4-full.bin</li>
    <li>ffmpeg_4-full.data</li>
    <li>ffmpeg_4-full.dev</li>
    <li>ffmpeg_4-full.doc</li>
    <li>ffmpeg_4-full.lib</li>
    <li>ffmpeg_4-full.man</li>
    <li>ffmpeg_6-full</li>
    <li>ffmpeg_6-full.bin</li>
    <li>ffmpeg_6-full.data</li>
    <li>ffmpeg_6-full.dev</li>
    <li>ffmpeg_6-full.doc</li>
    <li>ffmpeg_6-full.lib</li>
    <li>ffmpeg_6-full.man</li>
    <li>flashfocus</li>
    <li>flashfocus.dist</li>
    <li>folks</li>
    <li>folks.dev</li>
    <li>folks.devdoc</li>
    <li>formatter</li>
    <li>freeipa</li>
    <li>fwts</li>
    <li>gfbgraph</li>
    <li>gfbgraph.dev</li>
    <li>gfbgraph.devdoc</li>
    <li>gnome-browser-connector</li>
    <li>gnome-online-accounts</li>
    <li>gnome-online-accounts.debug</li>
    <li>gnome-online-accounts.dev</li>
    <li>gnome-online-accounts.devdoc</li>
    <li>gnome-online-accounts.man</li>
    <li>gnome-photos</li>
    <li>gnome-photos.installedTests</li>
    <li>gnome-recipes</li>
    <li>gnome.cheese</li>
    <li>gnome.cheese.devdoc</li>
    <li>gnome.cheese.man</li>
    <li>gnome.geary</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-boxes</li>
    <li>gnome.gnome-calendar</li>
    <li>gnome.gnome-contacts</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-control-center.debug</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-notes</li>
    <li>gnome.gnome-online-miners</li>
    <li>gnome.gnome-panel</li>
    <li>gnome.gnome-panel.dev</li>
    <li>gnome.gnome-panel.man</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-session.debug</li>
    <li>gnome.gnome-session.sessions</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-shell.debug</li>
    <li>gnome.gnome-shell.devdoc</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gvfs</li>
    <li>gnome.gvfs.debug</li>
    <li>gnome.totem</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.gsconnect.installedTests</li>
    <li>got</li>
    <li>grilo-plugins</li>
    <li>gscreenshot</li>
    <li>gscreenshot.dist</li>
    <li>gvfs</li>
    <li>gvfs.debug</li>
    <li>handbrake</li>
    <li>haskellPackages.libvirt-hs</li>
    <li>haskellPackages.libvirt-hs.doc</li>
    <li>health-check</li>
    <li>hfsprogs</li>
    <li>holochain-launcher</li>
    <li>hopper</li>
    <li>hydra_unstable</li>
    <li>i3lock-fancy</li>
    <li>i3lock-pixeled</li>
    <li>imagination</li>
    <li>imgur-screenshot</li>
    <li>inputplug</li>
    <li>jellyfin</li>
    <li>jibri</li>
    <li>jicofo</li>
    <li>jitsi-meet-prosody</li>
    <li>jitsi-videobridge</li>
    <li>kaffeine</li>
    <li>kamid</li>
    <li>kodelife</li>
    <li>kodi (kodiPackages.kodi)</li>
    <li>kodi-gbm</li>
    <li>kodi-wayland</li>
    <li>kodiPackages.inputstream-adaptive</li>
    <li>kodiPackages.inputstream-ffmpegdirect</li>
    <li>kodiPackages.inputstream-rtmp</li>
    <li>kodiPackages.joystick</li>
    <li>kodiPackages.libretro</li>
    <li>kodiPackages.libretro-2048</li>
    <li>kodiPackages.libretro-fuse</li>
    <li>kodiPackages.libretro-genplus</li>
    <li>kodiPackages.libretro-mgba</li>
    <li>kodiPackages.libretro-snes9x</li>
    <li>kodiPackages.pvr-hdhomerun</li>
    <li>kodiPackages.pvr-hts</li>
    <li>kodiPackages.pvr-iptvsimple</li>
    <li>kodiPackages.steam-controller</li>
    <li>kodiPackages.vfs-libarchive</li>
    <li>kodiPackages.vfs-rar</li>
    <li>kodiPackages.vfs-sftp</li>
    <li>kodiPackages.visualization-fishbmc</li>
    <li>kodiPackages.visualization-goom</li>
    <li>kodiPackages.visualization-matrix</li>
    <li>kodiPackages.visualization-pictureit</li>
    <li>kodiPackages.visualization-projectm</li>
    <li>kodiPackages.visualization-shadertoy</li>
    <li>kodiPackages.visualization-spectrum</li>
    <li>kodiPackages.visualization-starburst</li>
    <li>kodiPackages.visualization-waveform</li>
    <li>koreader</li>
    <li>kphotoalbum</li>
    <li>libbsd</li>
    <li>libbsd.dev</li>
    <li>libbsd.man</li>
    <li>libgdata</li>
    <li>libgdata.dev</li>
    <li>libgdata.installedTests</li>
    <li>libguestfs</li>
    <li>libmd</li>
    <li>libsForQt5.elisa (plasma5Packages.elisa)</li>
    <li>libsForQt5.elisa.dev (plasma5Packages.elisa.dev)</li>
    <li>libsForQt5.kdenetwork-filesharing (plasma5Packages.kdenetwork-filesharing)</li>
    <li>libsForQt5.kdenlive (plasma5Packages.kdenlive)</li>
    <li>libsForQt5.kio-extras (plasma5Packages.kio-extras)</li>
    <li>libsForQt5.phonon-backend-vlc (plasma5Packages.phonon-backend-vlc)</li>
    <li>libvirt</li>
    <li>libvirt-glib</li>
    <li>libvirt-glib.dev</li>
    <li>libvirt-glib.devdoc</li>
    <li>libvlc</li>
    <li>libzapojit</li>
    <li>libzapojit.dev</li>
    <li>odp-dpdk (linuxPackages.odp-dpdk)</li>
    <li>linuxKernel.packages.linux_4_14.dpdk</li>
    <li>linuxKernel.packages.linux_4_14.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_14.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_4_19.dpdk</li>
    <li>linuxKernel.packages.linux_4_19.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_19.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_10.dpdk</li>
    <li>linuxKernel.packages.linux_5_10.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_10.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_15.dpdk</li>
    <li>linuxKernel.packages.linux_5_15.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_15.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_4.dpdk</li>
    <li>linuxKernel.packages.linux_5_4.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_4.dpdk.kmod</li>
    <li>linuxPackages.dpdk (linuxKernel.packages.linux_6_1.dpdk)</li>
    <li>linuxPackages.dpdk.doc (linuxKernel.packages.linux_6_1.dpdk.doc)</li>
    <li>linuxPackages.dpdk.kmod (linuxKernel.packages.linux_6_1.dpdk.kmod)</li>
    <li>linuxPackages-libre.dpdk (linuxKernel.packages.linux_libre.dpdk)</li>
    <li>linuxPackages-libre.dpdk.doc (linuxKernel.packages.linux_libre.dpdk.doc)</li>
    <li>linuxPackages-libre.dpdk.kmod (linuxKernel.packages.linux_libre.dpdk.kmod)</li>
    <li>liquidsoap</li>
    <li>lxd</li>
    <li>manim</li>
    <li>manim.dist</li>
    <li>marvin</li>
    <li>megaglest</li>
    <li>metastore</li>
    <li>mgmt</li>
    <li>minikube</li>
    <li>minishift</li>
    <li>minitube</li>
    <li>mlxbf-bootimages</li>
    <li>mojave-gtk-theme</li>
    <li>molly-guard</li>
    <li>monado</li>
    <li>mpd</li>
    <li>mpd.doc</li>
    <li>mpd.man</li>
    <li>mullvad-vpn</li>
    <li>netcat-openbsd</li>
    <li>networkaudiod</li>
    <li>nixops_unstable</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.advanced-scene-switcher</li>
    <li>obs-studio-plugins.droidcam-obs</li>
    <li>obs-studio-plugins.obs-livesplit-one</li>
    <li>obs-studio-plugins.obs-teleport</li>
    <li>obs-studio-plugins.obs-tuna</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>opendkim</li>
    <li>opentabletdriver</li>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-mail</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-tasks</li>
    <li>pantheon.switchboard-plug-onlineaccounts</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>parsec-bin</li>
    <li>pdfstudio2021</li>
    <li>pdfstudio2022</li>
    <li>pdisk</li>
    <li>perl534Packages.SysVirt</li>
    <li>perl534Packages.SysVirt.devdoc</li>
    <li>perl536Packages.SysVirt</li>
    <li>perl536Packages.SysVirt.devdoc</li>
    <li>phosh</li>
    <li>phosh-mobile-settings</li>
    <li>pkg</li>
    <li>pkg.debug</li>
    <li>planify</li>
    <li>plex</li>
    <li>plexRaw</li>
    <li>plexRaw.basedb</li>
    <li>podman</li>
    <li>podman.man</li>
    <li>polar-bookshelf</li>
    <li>polar-bookshelf1</li>
    <li>printrun</li>
    <li>printrun.dist</li>
    <li>pympress</li>
    <li>pympress.dist</li>
    <li>python310Packages.aigpy</li>
    <li>python310Packages.aigpy.dist</li>
    <li>python310Packages.django-scim2</li>
    <li>python310Packages.django-scim2.dist</li>
    <li>python310Packages.guestfs</li>
    <li>python310Packages.guestfs.dist</li>
    <li>python310Packages.libvirt</li>
    <li>python310Packages.libvirt.dist</li>
    <li>python310Packages.moderngl-window</li>
    <li>python310Packages.moderngl-window.dist</li>
    <li>python310Packages.py-libzfs</li>
    <li>python310Packages.py-libzfs.dist</li>
    <li>python310Packages.pyautogui</li>
    <li>python310Packages.pyautogui.dist</li>
    <li>python310Packages.pydub</li>
    <li>python310Packages.pydub.dist</li>
    <li>python310Packages.pyglet</li>
    <li>python310Packages.pyglet.dist</li>
    <li>python310Packages.pyscreeze</li>
    <li>python310Packages.pyscreeze.dist</li>
    <li>python310Packages.pysmbc</li>
    <li>python310Packages.pysmbc.dist</li>
    <li>python310Packages.python-vlc</li>
    <li>python310Packages.python-vlc.dist</li>
    <li>python310Packages.pytmx</li>
    <li>python310Packages.pytmx.dist</li>
    <li>python311Packages.aigpy</li>
    <li>python311Packages.aigpy.dist</li>
    <li>python311Packages.django-scim2</li>
    <li>python311Packages.django-scim2.dist</li>
    <li>python311Packages.guestfs</li>
    <li>python311Packages.guestfs.dist</li>
    <li>python311Packages.libvirt</li>
    <li>python311Packages.libvirt.dist</li>
    <li>python311Packages.moderngl-window</li>
    <li>python311Packages.moderngl-window.dist</li>
    <li>python311Packages.py-libzfs</li>
    <li>python311Packages.py-libzfs.dist</li>
    <li>python311Packages.pyautogui</li>
    <li>python311Packages.pyautogui.dist</li>
    <li>python311Packages.pydub</li>
    <li>python311Packages.pydub.dist</li>
    <li>python311Packages.pyglet</li>
    <li>python311Packages.pyglet.dist</li>
    <li>python311Packages.pyscreeze</li>
    <li>python311Packages.pyscreeze.dist</li>
    <li>python311Packages.pysmbc</li>
    <li>python311Packages.pysmbc.dist</li>
    <li>python311Packages.python-vlc</li>
    <li>python311Packages.python-vlc.dist</li>
    <li>python311Packages.pytmx</li>
    <li>python311Packages.pytmx.dist</li>
    <li>qq</li>
    <li>quickgui</li>
    <li>rabbitvcs</li>
    <li>rabbitvcs.cli</li>
    <li>rabbitvcs.dist</li>
    <li>radiotray-ng</li>
    <li>rcon</li>
    <li>reaper</li>
    <li>rofi-screenshot</li>
    <li>rosie</li>
    <li>rs</li>
    <li>rubyPackages.ruby-libvirt (rubyPackages_3_1.ruby-libvirt)</li>
    <li>rubyPackages_3_0.ruby-libvirt</li>
    <li>rubyPackages_3_2.ruby-libvirt</li>
    <li>rubyPackages_3_3.ruby-libvirt</li>
    <li>samba (samba4)</li>
    <li>samba.dev (samba4.dev)</li>
    <li>samba.man (samba4.man)</li>
    <li>sambaFull (samba4Full)</li>
    <li>sambaFull.dev (samba4Full.dev)</li>
    <li>sambaFull.man (samba4Full.man)</li>
    <li>sanoid</li>
    <li>scrot</li>
    <li>signify</li>
    <li>signing-party</li>
    <li>smbnetfs</li>
    <li>snis</li>
    <li>squeekboard</li>
    <li>sssd</li>
    <li>standardnotes</li>
    <li>strawberry</li>
    <li>stress-ng</li>
    <li>sunshine</li>
    <li>system-config-printer</li>
    <li>tidal-dl</li>
    <li>tidal-dl.dist</li>
    <li>tone</li>
    <li>touchosc</li>
    <li>unifi (unifi7)</li>
    <li>unifi5</li>
    <li>unifi6</li>
    <li>unifiLTS</li>
    <li>unixODBCDrivers.msodbcsql17</li>
    <li>unixODBCDrivers.redshift</li>
    <li>ustreamer</li>
    <li>vagrant</li>
    <li>valent</li>
    <li>video-trimmer</li>
    <li>vimix-gtk-themes</li>
    <li>virt-manager</li>
    <li>virt-manager-qt</li>
    <li>virt-manager.dist</li>
    <li>virt-top</li>
    <li>virt-viewer</li>
    <li>vlc</li>
    <li>vokoscreen</li>
    <li>whitesur-gtk-theme</li>
    <li>yesplaymusic</li>
    <li>zfs (zfsStable ,zfsUnstable)</li>
    <li>zfs.dev (zfsStable.dev ,zfsUnstable.dev)</li>
    <li>zfstools</li>
    <li>zpool-auto-expand-partitions</li>
  </ul>
</details>

---

Result of `nixpkgs-review pr 247082` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>74 packages marked as broken and skipped:</summary>
  <ul>
    <li>aws-workspaces</li>
    <li>binance</li>
    <li>cinny-desktop</li>
    <li>clockify</li>
    <li>etcher</li>
    <li>fahcontrol</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>googleearth-pro</li>
    <li>ideamaker</li>
    <li>itch</li>
    <li>libvmi</li>
    <li>linuxKernel.packages.hardkernel_4_14.dpdk</li>
    <li>linuxKernel.packages.hardkernel_4_14.dpdk.doc</li>
    <li>linuxKernel.packages.hardkernel_4_14.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_4_14_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_4_14_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_14_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_4_19_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_4_19_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_19_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_10_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_5_10_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_10_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_15_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_5_15_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_15_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_4_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_5_4_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_4_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_6_1_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_6_1_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_6_1_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_6_4_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_6_4_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_6_4_hardened.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_hardened.dpdk</li>
    <li>linuxKernel.packages.linux_hardened.dpdk.doc</li>
    <li>linuxKernel.packages.linux_hardened.dpdk.kmod</li>
    <li>linuxPackages_4_14_hardened.dpdk</li>
    <li>linuxPackages_4_14_hardened.dpdk.doc</li>
    <li>linuxPackages_4_14_hardened.dpdk.kmod</li>
    <li>linuxPackages_4_19_hardened.dpdk</li>
    <li>linuxPackages_4_19_hardened.dpdk.doc</li>
    <li>linuxPackages_4_19_hardened.dpdk.kmod</li>
    <li>linuxPackages_5_10_hardened.dpdk</li>
    <li>linuxPackages_5_10_hardened.dpdk.doc</li>
    <li>linuxPackages_5_10_hardened.dpdk.kmod</li>
    <li>linuxPackages_5_15_hardened.dpdk</li>
    <li>linuxPackages_5_15_hardened.dpdk.doc</li>
    <li>linuxPackages_5_15_hardened.dpdk.kmod</li>
    <li>linuxPackages_5_4_hardened.dpdk</li>
    <li>linuxPackages_5_4_hardened.dpdk.doc</li>
    <li>linuxPackages_5_4_hardened.dpdk.kmod</li>
    <li>linuxPackages_6_1_hardened.dpdk</li>
    <li>linuxPackages_6_1_hardened.dpdk.doc</li>
    <li>linuxPackages_6_1_hardened.dpdk.kmod</li>
    <li>linuxPackages_6_4_hardened.dpdk</li>
    <li>linuxPackages_6_4_hardened.dpdk.doc</li>
    <li>linuxPackages_6_4_hardened.dpdk.kmod</li>
    <li>linuxPackages_hardened.dpdk</li>
    <li>linuxPackages_hardened.dpdk.doc</li>
    <li>linuxPackages_hardened.dpdk.kmod</li>
    <li>linuxPackages_hardkernel_latest.dpdk</li>
    <li>linuxPackages_hardkernel_latest.dpdk.doc</li>
    <li>linuxPackages_hardkernel_latest.dpdk.kmod</li>
    <li>mailnagWithPlugins</li>
    <li>mailnagWithPlugins.dist</li>
    <li>obs-studio-plugins.obs-hyperion</li>
    <li>pcsc-safenet</li>
    <li>rubyPackages_2_7.ruby-libvirt</li>
    <li>runescape</li>
    <li>unifi-video</li>
    <li>viber</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>31 packages failed to build:</summary>
  <ul>
    <li>ciscoPacketTracer7</li>
    <li>ciscoPacketTracer8</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>fxlinuxprint</li>
    <li>fxlinuxprintutil</li>
    <li>hydra_unstable</li>
    <li>linuxPackages_latest.dpdk (linuxKernel.packages.linux_6_4.dpdk)</li>
    <li>linuxPackages_latest.dpdk.doc (linuxKernel.packages.linux_6_4.dpdk.doc)</li>
    <li>linuxPackages_latest.dpdk.kmod (linuxKernel.packages.linux_6_4.dpdk.kmod)</li>
    <li>linuxPackages_latest-libre.dpdk (linuxKernel.packages.linux_latest_libre.dpdk)</li>
    <li>linuxPackages_latest-libre.dpdk.doc (linuxKernel.packages.linux_latest_libre.dpdk.doc)</li>
    <li>linuxPackages_latest-libre.dpdk.kmod (linuxKernel.packages.linux_latest_libre.dpdk.kmod)</li>
    <li>linuxPackages_lqx.dpdk (linuxKernel.packages.linux_lqx.dpdk)</li>
    <li>linuxPackages_lqx.dpdk.doc (linuxKernel.packages.linux_lqx.dpdk.doc)</li>
    <li>linuxPackages_lqx.dpdk.kmod (linuxKernel.packages.linux_lqx.dpdk.kmod)</li>
    <li>linuxPackages_testing_bcachefs.dpdk (linuxKernel.packages.linux_testing_bcachefs.dpdk)</li>
    <li>linuxPackages_testing_bcachefs.dpdk.doc (linuxKernel.packages.linux_testing_bcachefs.dpdk.doc)</li>
    <li>linuxPackages_testing_bcachefs.dpdk.kmod (linuxKernel.packages.linux_testing_bcachefs.dpdk.kmod)</li>
    <li>linuxPackages_xanmod_latest.dpdk (linuxKernel.packages.linux_xanmod_latest.dpdk ,linuxPackages_xanmod_stable.dpdk)</li>
    <li>linuxPackages_xanmod_latest.dpdk.doc (linuxKernel.packages.linux_xanmod_latest.dpdk.doc ,linuxPackages_xanmod_stable.dpdk.doc)</li>
    <li>linuxPackages_xanmod_latest.dpdk.kmod (linuxKernel.packages.linux_xanmod_latest.dpdk.kmod ,linuxPackages_xanmod_stable.dpdk.kmod)</li>
    <li>linuxPackages_zen.dpdk (linuxKernel.packages.linux_zen.dpdk)</li>
    <li>linuxPackages_zen.dpdk.doc (linuxKernel.packages.linux_zen.dpdk.doc)</li>
    <li>linuxPackages_zen.dpdk.kmod (linuxKernel.packages.linux_zen.dpdk.kmod)</li>
    <li>ocenaudio</li>
    <li>pdfstudioviewer</li>
    <li>redpanda-server</li>
    <li>signumone-ks</li>
    <li>upwork</li>
  </ul>
</details>
<details>
  <summary>658 packages built:</summary>
  <ul>
    <li>MIDIVisualizer</li>
    <li>acme-client</li>
    <li>adapta-gtk-theme</li>
    <li>alfaview</li>
    <li>almanah</li>
    <li>apkg</li>
    <li>apkg.dist</li>
    <li>appgate-sdp</li>
    <li>appvm</li>
    <li>apt</li>
    <li>arcanPackages.all-wrapped</li>
    <li>arcanPackages.arcan</li>
    <li>arcanPackages.arcan-wrapped</li>
    <li>arcanPackages.cat9-wrapped</li>
    <li>arcanPackages.durden-wrapped</li>
    <li>ffmpeg-full (arcanPackages.ffmpeg ,ffmpeg_5-full)</li>
    <li>ffmpeg-full.bin (arcanPackages.ffmpeg.bin ,ffmpeg_5-full.bin)</li>
    <li>ffmpeg-full.data (arcanPackages.ffmpeg.data ,ffmpeg_5-full.data)</li>
    <li>ffmpeg-full.dev (arcanPackages.ffmpeg.dev ,ffmpeg_5-full.dev)</li>
    <li>ffmpeg-full.doc (arcanPackages.ffmpeg.doc ,ffmpeg_5-full.doc)</li>
    <li>ffmpeg-full.lib (arcanPackages.ffmpeg.lib ,ffmpeg_5-full.lib)</li>
    <li>ffmpeg-full.man (arcanPackages.ffmpeg.man ,ffmpeg_5-full.man)</li>
    <li>arcanPackages.pipeworld-wrapped</li>
    <li>arcanPackages.prio-wrapped</li>
    <li>arcanPackages.xarcan</li>
    <li>armcord</li>
    <li>atlauncher</li>
    <li>aucatctl</li>
    <li>aucatctl.man</li>
    <li>audiobookshelf</li>
    <li>autofs5</li>
    <li>bdf2psf</li>
    <li>beyond-identity</li>
    <li>bisq-desktop</li>
    <li>bitscope.chart</li>
    <li>bitscope.console</li>
    <li>bitscope.display</li>
    <li>bitscope.dso</li>
    <li>bitscope.logic</li>
    <li>bitscope.meter</li>
    <li>bitscope.proto</li>
    <li>bitscope.server</li>
    <li>bitwig-studio (bitwig-studio5)</li>
    <li>bitwig-studio1</li>
    <li>bitwig-studio2</li>
    <li>bitwig-studio3</li>
    <li>bitwig-studio4</li>
    <li>blackfire</li>
    <li>bluemail</li>
    <li>bluez-alsa</li>
    <li>booster</li>
    <li>bottles</li>
    <li>brave</li>
    <li>breitbandmessung</li>
    <li>bubblemail</li>
    <li>bubblemail.dist</li>
    <li>budgie.budgie-control-center</li>
    <li>budgie.budgie-control-center.debug</li>
    <li>buildah</li>
    <li>buildah.man</li>
    <li>bumblebee</li>
    <li>calls</li>
    <li>calls.devdoc</li>
    <li>cargo-deb</li>
    <li>ceph (ceph-dev)</li>
    <li>ceph-client</li>
    <li>ceph-csi</li>
    <li>ceph.dev (ceph-dev.dev)</li>
    <li>ceph.doc (ceph-dev.doc)</li>
    <li>libceph (ceph.lib ,libceph.dev ,libceph.doc ,libceph.lib ,libceph.man)</li>
    <li>ceph.man (ceph-dev.man)</li>
    <li>chatty</li>
    <li>check_zfs</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-gsettings-overrides</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>cinnamon.nemo</li>
    <li>cinnamon.nemo-fileroller</li>
    <li>cinnamon.nemo-python</li>
    <li>cinnamon.nemo-with-extensions</li>
    <li>cinnamon.nemo.dev</li>
    <li>clash-verge</li>
    <li>clementine</li>
    <li>cloudflare-warp</li>
    <li>collectd</li>
    <li>corrscope</li>
    <li>corrscope.dist</li>
    <li>cri-o</li>
    <li>cri-o.man</li>
    <li>criu</li>
    <li>criu.dev</li>
    <li>criu.man</li>
    <li>crun</li>
    <li>csmith</li>
    <li>cups-brother-hl1110</li>
    <li>cups-brother-hl1210w</li>
    <li>cups-brother-hl3140cw</li>
    <li>cups-brother-hll2340dw</li>
    <li>cups-brother-hll2350dw</li>
    <li>cups-brother-hll2375dw</li>
    <li>cups-brother-hll3230cdw</li>
    <li>cups-brother-mfcl2750dw</li>
    <li>darling</li>
    <li>darling.sdk</li>
    <li>dcp9020cdw-cupswrapper</li>
    <li>dcp9020cdwlpr</li>
    <li>debian-devscripts</li>
    <li>debootstrap</li>
    <li>deepin.deepin-music</li>
    <li>deepin.deepin-voice-note</li>
    <li>distrobuilder</li>
    <li>docker-machine-kvm</li>
    <li>docker-machine-kvm2</li>
    <li>donkey</li>
    <li>dpdk</li>
    <li>dpdk.doc</li>
    <li>dpkg</li>
    <li>eaglemode</li>
    <li>easysnap</li>
    <li>edgetpu-compiler</li>
    <li>eid-mw</li>
    <li>endeavour</li>
    <li>enpass</li>
    <li>enum4linux</li>
    <li>enum4linux-ng</li>
    <li>enum4linux-ng.dist</li>
    <li>epson-alc1100</li>
    <li>evolution</li>
    <li>evolution-data-server (gnome.evolution-data-server)</li>
    <li>evolution-data-server-gtk4</li>
    <li>evolution-data-server-gtk4.dev</li>
    <li>evolution-data-server.dev (gnome.evolution-data-server.dev)</li>
    <li>evolution-ews</li>
    <li>evolutionWithPlugins</li>
    <li>exactaudiocopy</li>
    <li>expressvpn</li>
    <li>fahclient</li>
    <li>fahviewer</li>
    <li>far2l</li>
    <li>feishu</li>
    <li>ffcast</li>
    <li>ffmpeg_4-full</li>
    <li>ffmpeg_4-full.bin</li>
    <li>ffmpeg_4-full.data</li>
    <li>ffmpeg_4-full.dev</li>
    <li>ffmpeg_4-full.doc</li>
    <li>ffmpeg_4-full.lib</li>
    <li>ffmpeg_4-full.man</li>
    <li>ffmpeg_6-full</li>
    <li>ffmpeg_6-full.bin</li>
    <li>ffmpeg_6-full.data</li>
    <li>ffmpeg_6-full.dev</li>
    <li>ffmpeg_6-full.doc</li>
    <li>ffmpeg_6-full.lib</li>
    <li>ffmpeg_6-full.man</li>
    <li>figma-linux</li>
    <li>flashfocus</li>
    <li>flashfocus.dist</li>
    <li>folks</li>
    <li>folks.dev</li>
    <li>folks.devdoc</li>
    <li>formatter</li>
    <li>freeipa</li>
    <li>fwts</li>
    <li>gamepad-tool</li>
    <li>gfbgraph</li>
    <li>gfbgraph.dev</li>
    <li>gfbgraph.devdoc</li>
    <li>gitter</li>
    <li>gnome-browser-connector</li>
    <li>gnome-online-accounts</li>
    <li>gnome-online-accounts.debug</li>
    <li>gnome-online-accounts.dev</li>
    <li>gnome-online-accounts.devdoc</li>
    <li>gnome-online-accounts.man</li>
    <li>gnome-photos</li>
    <li>gnome-photos.installedTests</li>
    <li>gnome-recipes</li>
    <li>gnome.cheese</li>
    <li>gnome.cheese.devdoc</li>
    <li>gnome.cheese.man</li>
    <li>gnome.geary</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-boxes</li>
    <li>gnome.gnome-calendar</li>
    <li>gnome.gnome-contacts</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-control-center.debug</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-notes</li>
    <li>gnome.gnome-online-miners</li>
    <li>gnome.gnome-panel</li>
    <li>gnome.gnome-panel.dev</li>
    <li>gnome.gnome-panel.man</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-session.debug</li>
    <li>gnome.gnome-session.sessions</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-shell.debug</li>
    <li>gnome.gnome-shell.devdoc</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gvfs</li>
    <li>gnome.gvfs.debug</li>
    <li>gnome.nixos-gsettings-overrides</li>
    <li>gnome.totem</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.gsconnect.installedTests</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>gohufont</li>
    <li>got</li>
    <li>grilo-plugins</li>
    <li>gscreenshot</li>
    <li>gscreenshot.dist</li>
    <li>guestfs-tools</li>
    <li>gvfs</li>
    <li>gvfs.debug</li>
    <li>hakuneko</li>
    <li>handbrake</li>
    <li>haruna</li>
    <li>haskellPackages.libvirt-hs</li>
    <li>haskellPackages.libvirt-hs.doc</li>
    <li>headset</li>
    <li>health-check</li>
    <li>heroic</li>
    <li>hfsprogs</li>
    <li>hll2390dw-cups</li>
    <li>holochain-launcher</li>
    <li>hopper</li>
    <li>hyper</li>
    <li>i3lock-fancy</li>
    <li>i3lock-pixeled</li>
    <li>imagination</li>
    <li>imgur-screenshot</li>
    <li>inputplug</li>
    <li>insomnia</li>
    <li>insync</li>
    <li>ipscan</li>
    <li>iqueue</li>
    <li>jellyfin</li>
    <li>jellyfin-ffmpeg</li>
    <li>jellyfin-ffmpeg.bin</li>
    <li>jellyfin-ffmpeg.data</li>
    <li>jellyfin-ffmpeg.dev</li>
    <li>jellyfin-ffmpeg.doc</li>
    <li>jellyfin-ffmpeg.lib</li>
    <li>jellyfin-ffmpeg.man</li>
    <li>jibri</li>
    <li>jicofo</li>
    <li>jitsi-meet-prosody</li>
    <li>jitsi-videobridge</li>
    <li>join-desktop</li>
    <li>kaffeine</li>
    <li>kamid</li>
    <li>kcgi</li>
    <li>keeweb</li>
    <li>kodelife</li>
    <li>kodi (kodiPackages.kodi)</li>
    <li>kodi-gbm</li>
    <li>kodi-wayland</li>
    <li>kodiPackages.a4ksubtitles</li>
    <li>kodiPackages.archive_tool</li>
    <li>kodiPackages.iagl</li>
    <li>kodiPackages.inputstream-adaptive</li>
    <li>kodiPackages.inputstream-ffmpegdirect</li>
    <li>kodiPackages.inputstream-rtmp</li>
    <li>kodiPackages.invidious</li>
    <li>kodiPackages.joystick</li>
    <li>kodiPackages.kodi-platform</li>
    <li>kodiPackages.libretro</li>
    <li>kodiPackages.libretro-2048</li>
    <li>kodiPackages.libretro-fuse</li>
    <li>kodiPackages.libretro-genplus</li>
    <li>kodiPackages.libretro-mgba</li>
    <li>kodiPackages.libretro-snes9x</li>
    <li>kodiPackages.netflix</li>
    <li>kodiPackages.pvr-hdhomerun</li>
    <li>kodiPackages.pvr-hts</li>
    <li>kodiPackages.pvr-iptvsimple</li>
    <li>kodiPackages.steam-controller</li>
    <li>kodiPackages.steam-launcher</li>
    <li>kodiPackages.vfs-libarchive</li>
    <li>kodiPackages.vfs-rar</li>
    <li>kodiPackages.vfs-sftp</li>
    <li>kodiPackages.visualization-fishbmc</li>
    <li>kodiPackages.visualization-goom</li>
    <li>kodiPackages.visualization-matrix</li>
    <li>kodiPackages.visualization-pictureit</li>
    <li>kodiPackages.visualization-projectm</li>
    <li>kodiPackages.visualization-shadertoy</li>
    <li>kodiPackages.visualization-spectrum</li>
    <li>kodiPackages.visualization-starburst</li>
    <li>kodiPackages.visualization-waveform</li>
    <li>koreader</li>
    <li>kphotoalbum</li>
    <li>krunvm</li>
    <li>libbsd</li>
    <li>libbsd.dev</li>
    <li>libbsd.man</li>
    <li>libgdata</li>
    <li>libgdata.dev</li>
    <li>libgdata.installedTests</li>
    <li>libguestfs</li>
    <li>libmd</li>
    <li>libsForQt5.elisa (plasma5Packages.elisa)</li>
    <li>libsForQt5.elisa.dev (plasma5Packages.elisa.dev)</li>
    <li>libsForQt5.kdenetwork-filesharing (plasma5Packages.kdenetwork-filesharing)</li>
    <li>libsForQt5.kdenlive (plasma5Packages.kdenlive)</li>
    <li>libsForQt5.kio-extras (plasma5Packages.kio-extras)</li>
    <li>libsForQt5.phonon-backend-vlc (plasma5Packages.phonon-backend-vlc)</li>
    <li>soundkonverter (libsForQt5.soundkonverter)</li>
    <li>libvirt</li>
    <li>libvirt-glib</li>
    <li>libvirt-glib.dev</li>
    <li>libvirt-glib.devdoc</li>
    <li>libvlc</li>
    <li>libzapojit</li>
    <li>libzapojit.dev</li>
    <li>lightworks</li>
    <li>odp-dpdk (linuxPackages.odp-dpdk)</li>
    <li>pktgen (linuxPackages.pktgen)</li>
    <li>linuxKernel.packages.linux_4_14.dpdk</li>
    <li>linuxKernel.packages.linux_4_14.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_14.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_4_19.dpdk</li>
    <li>linuxKernel.packages.linux_4_19.dpdk.doc</li>
    <li>linuxKernel.packages.linux_4_19.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_10.dpdk</li>
    <li>linuxKernel.packages.linux_5_10.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_10.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_15.dpdk</li>
    <li>linuxKernel.packages.linux_5_15.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_15.dpdk.kmod</li>
    <li>linuxKernel.packages.linux_5_4.dpdk</li>
    <li>linuxKernel.packages.linux_5_4.dpdk.doc</li>
    <li>linuxKernel.packages.linux_5_4.dpdk.kmod</li>
    <li>linuxPackages.dpdk (linuxKernel.packages.linux_6_1.dpdk)</li>
    <li>linuxPackages.dpdk.doc (linuxKernel.packages.linux_6_1.dpdk.doc)</li>
    <li>linuxPackages.dpdk.kmod (linuxKernel.packages.linux_6_1.dpdk.kmod)</li>
    <li>linuxPackages-libre.dpdk (linuxKernel.packages.linux_libre.dpdk)</li>
    <li>linuxPackages-libre.dpdk.doc (linuxKernel.packages.linux_libre.dpdk.doc)</li>
    <li>linuxPackages-libre.dpdk.kmod (linuxKernel.packages.linux_libre.dpdk.kmod)</li>
    <li>linuxPackages_xanmod.dpdk (linuxKernel.packages.linux_xanmod.dpdk)</li>
    <li>linuxPackages_xanmod.dpdk.doc (linuxKernel.packages.linux_xanmod.dpdk.doc)</li>
    <li>linuxPackages_xanmod.dpdk.kmod (linuxKernel.packages.linux_xanmod.dpdk.kmod)</li>
    <li>liquidsoap</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>lxd</li>
    <li>mailspring</li>
    <li>manim</li>
    <li>manim.dist</li>
    <li>marvin</li>
    <li>mblock-mlink</li>
    <li>megaglest</li>
    <li>metastore</li>
    <li>mfc5890cncupswrapper</li>
    <li>mfc5890cnlpr</li>
    <li>mfc9140cdncupswrapper</li>
    <li>mfc9140cdnlpr</li>
    <li>mfcj470dw-cupswrapper</li>
    <li>mfcj470dwlpr</li>
    <li>mfcj6510dw-cupswrapper</li>
    <li>mfcj6510dwlpr</li>
    <li>mfcl2700dncupswrapper</li>
    <li>mfcl2720dwcupswrapper</li>
    <li>mfcl2720dwlpr</li>
    <li>mfcl2740dwcupswrapper</li>
    <li>mfcl2740dwlpr</li>
    <li>mfcl3770cdwcupswrapper</li>
    <li>mfcl3770cdwlpr</li>
    <li>mfcl8690cdwcupswrapper</li>
    <li>mfcl8690cdwlpr</li>
    <li>mgmt</li>
    <li>minigalaxy</li>
    <li>minigalaxy.dist</li>
    <li>minikube</li>
    <li>minishift</li>
    <li>minitube</li>
    <li>mojave-gtk-theme</li>
    <li>molly-guard</li>
    <li>monado</li>
    <li>mongodb-compass</li>
    <li>morgen</li>
    <li>mpd</li>
    <li>mpd.doc</li>
    <li>mpd.man</li>
    <li>mullvad-vpn</li>
    <li>multipass</li>
    <li>multiviewer-for-f1</li>
    <li>netcat-openbsd</li>
    <li>networkaudiod</li>
    <li>nixops_unstable</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.advanced-scene-switcher</li>
    <li>obs-studio-plugins.droidcam-obs</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-3d-effect</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>obs-studio-plugins.obs-command-source</li>
    <li>obs-studio-plugins.obs-gradient-source</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-livesplit-one</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-mute-filter</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-rgb-levels-filter</li>
    <li>obs-studio-plugins.obs-scale-to-sound</li>
    <li>obs-studio-plugins.obs-shaderfilter</li>
    <li>obs-studio-plugins.obs-source-clone</li>
    <li>obs-studio-plugins.obs-source-record</li>
    <li>obs-studio-plugins.obs-source-switcher</li>
    <li>obs-studio-plugins.obs-teleport</li>
    <li>obs-studio-plugins.obs-text-pthread</li>
    <li>obs-studio-plugins.obs-transition-table</li>
    <li>obs-studio-plugins.obs-tuna</li>
    <li>obs-studio-plugins.obs-vaapi</li>
    <li>obs-studio-plugins.obs-vertical-canvas</li>
    <li>obs-studio-plugins.obs-vintage-filter</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.obs-websocket</li>
    <li>obs-studio-plugins.waveform</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>odafileconverter</li>
    <li>onlyoffice-bin</li>
    <li>onlyoffice-documentserver</li>
    <li>opendkim</li>
    <li>opentabletdriver</li>
    <li>opera</li>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-mail</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-tasks</li>
    <li>pantheon.switchboard-plug-onlineaccounts</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>pantum-driver</li>
    <li>parsec-bin</li>
    <li>partition-manager</li>
    <li>pdfstudio2021</li>
    <li>pdfstudio2022</li>
    <li>pdisk</li>
    <li>peek</li>
    <li>perl534Packages.SysVirt</li>
    <li>perl534Packages.SysVirt.devdoc</li>
    <li>perl536Packages.SysVirt</li>
    <li>perl536Packages.SysVirt.devdoc</li>
    <li>phosh</li>
    <li>phosh-mobile-settings</li>
    <li>php81Extensions.smbclient</li>
    <li>php82Extensions.smbclient</li>
    <li>php83Extensions.smbclient</li>
    <li>picoscope</li>
    <li>pipelight</li>
    <li>pixeluvo</li>
    <li>pkg</li>
    <li>pkg.debug</li>
    <li>planify</li>
    <li>playonlinux</li>
    <li>plex</li>
    <li>plexRaw</li>
    <li>plexRaw.basedb</li>
    <li>podman</li>
    <li>podman.man</li>
    <li>polar-bookshelf</li>
    <li>polar-bookshelf1</li>
    <li>printrun</li>
    <li>printrun.dist</li>
    <li>protontricks</li>
    <li>protontricks.dist</li>
    <li>pympress</li>
    <li>pympress.dist</li>
    <li>python310Packages.aigpy</li>
    <li>python310Packages.aigpy.dist</li>
    <li>python310Packages.django-scim2</li>
    <li>python310Packages.django-scim2.dist</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python310Packages.dm-sonnet.dist</li>
    <li>python310Packages.guestfs</li>
    <li>python310Packages.guestfs.dist</li>
    <li>python310Packages.libvirt</li>
    <li>python310Packages.libvirt.dist</li>
    <li>python310Packages.moderngl-window</li>
    <li>python310Packages.moderngl-window.dist</li>
    <li>python310Packages.py-libzfs</li>
    <li>python310Packages.py-libzfs.dist</li>
    <li>python310Packages.pyautogui</li>
    <li>python310Packages.pyautogui.dist</li>
    <li>python310Packages.pydub</li>
    <li>python310Packages.pydub.dist</li>
    <li>python310Packages.pyglet</li>
    <li>python310Packages.pyglet.dist</li>
    <li>python310Packages.pyscreeze</li>
    <li>python310Packages.pyscreeze.dist</li>
    <li>python310Packages.pysmbc</li>
    <li>python310Packages.pysmbc.dist</li>
    <li>python310Packages.python-vlc</li>
    <li>python310Packages.python-vlc.dist</li>
    <li>python310Packages.pytmx</li>
    <li>python310Packages.pytmx.dist</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python310Packages.tensorflow-datasets.dist</li>
    <li>python311Packages.aigpy</li>
    <li>python311Packages.aigpy.dist</li>
    <li>python311Packages.django-scim2</li>
    <li>python311Packages.django-scim2.dist</li>
    <li>python311Packages.guestfs</li>
    <li>python311Packages.guestfs.dist</li>
    <li>python311Packages.libvirt</li>
    <li>python311Packages.libvirt.dist</li>
    <li>python311Packages.moderngl-window</li>
    <li>python311Packages.moderngl-window.dist</li>
    <li>python311Packages.py-libzfs</li>
    <li>python311Packages.py-libzfs.dist</li>
    <li>python311Packages.pyautogui</li>
    <li>python311Packages.pyautogui.dist</li>
    <li>python311Packages.pydub</li>
    <li>python311Packages.pydub.dist</li>
    <li>python311Packages.pyglet</li>
    <li>python311Packages.pyglet.dist</li>
    <li>python311Packages.pyscreeze</li>
    <li>python311Packages.pyscreeze.dist</li>
    <li>python311Packages.pysmbc</li>
    <li>python311Packages.pysmbc.dist</li>
    <li>python311Packages.python-vlc</li>
    <li>python311Packages.python-vlc.dist</li>
    <li>python311Packages.pytmx</li>
    <li>python311Packages.pytmx.dist</li>
    <li>q4wine</li>
    <li>qemu_full</li>
    <li>qemu_full.debug</li>
    <li>qemu_full.ga</li>
    <li>qq</li>
    <li>quickgui</li>
    <li>rabbitvcs</li>
    <li>rabbitvcs.cli</li>
    <li>rabbitvcs.dist</li>
    <li>radiotray-ng</li>
    <li>rcon</li>
    <li>reaper</li>
    <li>rescuetime</li>
    <li>restream</li>
    <li>roam-research</li>
    <li>rofi-screenshot</li>
    <li>rosie</li>
    <li>rs</li>
    <li>rubyPackages.ruby-libvirt (rubyPackages_3_1.ruby-libvirt)</li>
    <li>rubyPackages_3_0.ruby-libvirt</li>
    <li>rubyPackages_3_2.ruby-libvirt</li>
    <li>rubyPackages_3_3.ruby-libvirt</li>
    <li>samba (samba4)</li>
    <li>samba.dev (samba4.dev)</li>
    <li>samba.man (samba4.man)</li>
    <li>sambaFull (samba4Full)</li>
    <li>sambaFull.dev (samba4Full.dev)</li>
    <li>sambaFull.man (samba4Full.man)</li>
    <li>sanoid</li>
    <li>scrot</li>
    <li>signal-desktop</li>
    <li>signal-desktop-beta</li>
    <li>signify</li>
    <li>signing-party</li>
    <li>simplenote</li>
    <li>skypeforlinux</li>
    <li>slack</li>
    <li>smbnetfs</li>
    <li>snis</li>
    <li>spdk</li>
    <li>squeekboard</li>
    <li>sslmate-agent</li>
    <li>sssd</li>
    <li>standardnotes</li>
    <li>staruml</li>
    <li>steam (steamPackages.steam-fhsenv)</li>
    <li>steam-run</li>
    <li>steam-tui</li>
    <li>steamPackages.steam-fhsenv-without-steam</li>
    <li>steamcmd (steamPackages.steamcmd)</li>
    <li>strawberry</li>
    <li>stress-ng</li>
    <li>sunshine</li>
    <li>synology-drive-client</li>
    <li>system-config-printer</li>
    <li>teams</li>
    <li>terra-station</li>
    <li>tetrio-desktop</li>
    <li>thedesk</li>
    <li>threema-desktop</li>
    <li>tidal-dl</li>
    <li>tidal-dl.dist</li>
    <li>tidal-hifi</li>
    <li>tone</li>
    <li>tonelib-gfx</li>
    <li>tonelib-jam</li>
    <li>tonelib-metal</li>
    <li>tonelib-noisereducer</li>
    <li>tonelib-zoom</li>
    <li>touchosc</li>
    <li>trillian-im</li>
    <li>twingate</li>
    <li>unifi (unifi7)</li>
    <li>unifi5</li>
    <li>unifi6</li>
    <li>unifiLTS</li>
    <li>unigine-superposition</li>
    <li>unityhub</li>
    <li>unixODBCDrivers.msodbcsql17</li>
    <li>unixODBCDrivers.redshift</li>
    <li>ustreamer</li>
    <li>uw-ttyp0</li>
    <li>vagrant</li>
    <li>valent</li>
    <li>video-trimmer</li>
    <li>vimix-gtk-themes</li>
    <li>virt-manager</li>
    <li>virt-manager-qt</li>
    <li>virt-manager.dist</li>
    <li>virt-top</li>
    <li>virt-viewer</li>
    <li>vkd3d</li>
    <li>vkd3d-proton</li>
    <li>vlc</li>
    <li>vokoscreen</li>
    <li>whitesur-gtk-theme</li>
    <li>wine (winePackages.full ,winePackages.stableFull)</li>
    <li>wine-staging (winePackages.stagingFull)</li>
    <li>wine-wayland (winePackages.waylandFull)</li>
    <li>wine64 (wine64Packages.full ,wine64Packages.stableFull)</li>
    <li>wine64Packages.stagingFull</li>
    <li>wine64Packages.unstableFull</li>
    <li>wine64Packages.waylandFull</li>
    <li>winePackages.unstableFull</li>
    <li>wineWow64Packages.full (wineWow64Packages.stableFull)</li>
    <li>wineWow64Packages.stagingFull</li>
    <li>wineWow64Packages.unstableFull</li>
    <li>wineWow64Packages.waylandFull</li>
    <li>wineWowPackages.full (wineWowPackages.stableFull)</li>
    <li>wineWowPackages.stagingFull</li>
    <li>wineWowPackages.unstableFull</li>
    <li>wineWowPackages.waylandFull</li>
    <li>wire-desktop</li>
    <li>wtwitch</li>
    <li>xivlauncher</li>
    <li>yesplaymusic</li>
    <li>zfs (zfsStable ,zfsUnstable)</li>
    <li>zfs.dev (zfsStable.dev ,zfsUnstable.dev)</li>
    <li>zfstools</li>
    <li>zpool-auto-expand-partitions</li>
    <li>zxfer</li>
  </ul>
</details>

emilazy commented:
Thank you so much for trying this out! Looks like the DPDK and Python failures are pre-existing, so Ceph (AArch64), QEMU (AArch64), diffoscope (x86-64) and Hydra (x86-64) are the main big regressions here. I'll try to look into those soon, but since some of those are quite large builds, could I trouble you to Gist the `nix log`s of the build failures if you have them to hand in case it's something obvious?

This should probably go to `staging` when it's ready, but I'll keep it on `master` for now for easier regression testing.

adamcstephens commented:
Hmm, a gist seems awkward for this many files so I put it in a repo. https://github.com/adamcstephens/nixpkgs-reviews/tree/main/pr247082

emilazy commented:
Thank you so much!

It's interesting to look at these. Ignoring the packages that are already failing to build on Hydra regardless, I couldn't find any failure that was clearly related to libsd. For the Ceph packages the C compiler got killed and that broke the build (maybe OOM killer / some system load issue?), the QEMU log is empty, the diffoscope failure is some weird encoding issue, and the Hydra one is some database issue.

I don't understand why the latter two happened for you but not on Hydra, but it's hard for me to imagine how this bump would be the cause of any of them. The vast majority of libbsd changes since the last release have been build system tweaks and portability changes for non-Linux platforms, and notably it seems like none of the "new" failures occur on both platforms, so I think that hopefully this might not cause any actual regressions?

I've retargeted this for staging and pushed a cleaner fix for the `progname.c` issue that I've also sent upstream. If the same failures @adamcstephens ran into reoccur on Hydra, I'm happy to look into them, but I think this should be good to merge. Hopefully I won't have to eat my words there!

emilazy commented:
(Although all that said, I don't actually know what the workflow is for checking for `staging` regressions with Hydra, so if anyone has more experience merging to `staging` than me I'd appreciate tips!)

jecaro reviewed (approved):
AFAIK inputplug is not intended to work on macOS. So I'm fine with that.
